content/docs/dvclive/api-reference/next_step.md

```diff
@@ -1,8 +1,8 @@
 # dvclive.next_step()
 
-Signals that the current step has ended. Check the
-[init parameters](/doc/dvclive/api-reference/init#parameters) for configuring
-the behavior.
+Signals that the current step has ended and increases `step` value by 1 (one).
+Check the [init parameters](/doc/dvclive/api-reference/init#parameters) for
```

Comment:
What is being configured about next_step behavior in init?

Reply:
I think this is explained in the description @dberenbaum but not sure the info is complete (I'd remove this sentence from the intro paragraph).